youtube_dl/extractor/liveleak.py

```diff
@@ -120,7 +120,17 @@ def _real_extract(self, url):
             }
 
         for idx, info_dict in enumerate(entries):
+            source_found = False
             for a_format in info_dict['formats']:
+                if not source_found:
+                    source_url = re.sub(r'(https?://(?:\w+\.)?liveleak\.com/.*?\.\w+)\.\w+\.mp4((?:\?.+)?)', r'\1\2', a_format['url'])
```

Comment:
Just replace the part appended to non source formats with empty string.

Reply:
Do you mean the URL parameters? They're required as some sort of temporary access key.